server.md

```diff
@@ -118,29 +117,12 @@ using the GitHub repo URL in its `package.json` file:
 }
 ```
 
-Because servers are not typically published to NPM, we must use git-based URLs 
-when defining out dependency along with a git tag for versioning. The server repo 
-must properly manage their version using git tags and follow 
-[semantic versioning](https://semver.org/).
+Because servers are not typically published to NPM, we must use git-based URLs when defining out dependency along with a git tag for versioning. The server repo must properly manage their version using git tags and follow [semantic versioning](https://semver.org/).
```

Comment:
Actually, the bit about server versioning is misleading

Because each server repo uses its version number to describe changes to its endpoints, semantic verioning does *not* apply to the server's exported cleaner functions or types. For instance, a server might rename a cleaner function from `asLoginResponse` to `asLoginPayload`, but leave the contents the same. Although this is a breaking change for anybody using the `asLoginResponse` cleaner, it does not change the server's version number, since the server has not changed any endpoints. Clients should therefore use git hashes to lock themselves to a specific server commit, and treat every hash update as a potentially breaking change.

Reply:
^ Perhaps the above paragraph should go near the top of the "Server types" section, along with a note that we encourage putting the shared types in the client library, since that library can change its version number freely in response to code changes like this. Putting types in the server repo will always be is a bit janky, but we allow it as a stopgap, since it's better t have types *somewhere* than not at all.